src/fragments/lib/datastore/js/data-access/observe-update-snippet.mdx

```diff
@@ -22,35 +107,59 @@ function App() {
     };
   }, []);
 
+  /**
+	 * Create a new Post
+	 */
+	async function onCreate() {
+		const post: Post = await DataStore.save<Post>(
+			new Post({
+				title: `New title ${Date.now()}`,
+				rating: (function getRandomInt(min, max) {
+					min = Math.ceil(min);
+					max = Math.floor(max);
+					return Math.floor(Math.random() * (max - min)) + min;
+				})(1, 7),
```

Comment:
This inline function will produce a bit of variety for this as an example, but why not just include the `Math.floor(Math.random() * ...)` inline?

Reply:
This TypeScript example was copied from the existing JS example. I've gone ahead and updated all instances in the docs.